src/plugins/discover/server/ui_settings.ts

```diff
@@ -124,6 +125,19 @@ export const getUiSettings: (docLinks: DocLinksServiceSetup) => Record<string, U
     category: ['discover'],
     schema: schema.boolean(),
   },
+  [SHOW_LEGACY_FIELD_TOP_VALUES]: {
+    name: i18n.translate('discover.advancedSettings.showLegacyFieldStatsTitle', {
+      defaultMessage: "Show legacy Top Values for a field in the sidebar's popover",
```

Comment:
This title is a little long.  Is this about the display of the popover or how the top values are calculated?  If the latter, how about:

> Top value calculation

Reply:
@gchaps I think it's worth highlighting that it affects stats calculations only inside the field popover. It does not change Field Statistics page for example.